fiftyone/core/state.py

```diff
@@ -161,7 +161,12 @@ def from_dict(cls, d, with_config=None):
         view_name = d.get("view_name", None)
         if dataset is not None:
             if view_name:
-                view = dataset.load_saved_view(view_name)
+                try:
+                    view = dataset.load_saved_view(view_name)
+                except Exception as e:
+                    dataset.reload()
```

Comment:
should we always just reload the dataset when it's loaded on ln 157 above? Or would this affect performance or other issues?

Reply:
exactly. the questions are
- Why do we need to reload the dataset
- Is reloading it anyways has any performance issues